api/src/templates/new_member.body.html

```diff
@@ -1,167 +1,194 @@
-
-<h3>Får jag göra en prototyp på Stockholm Makerspace?</h3>
-<p>Ja, i mån av plats och om du inte säljer prototypen. Du ska heller inte ha bolagiserat ännu. Stockholm Makerspace är en ideell förening. För att kunna fortsätta vara det så får vi inte bedriva kommersiell verksamhet. Det betyder att du inte får göra produkter till försäljning.</p>
-
-<h3>Är det sant att jag får ha min alldeles egen låda här?</h3>
-<p>Yes! Köp din egen förvaringsbox med måtten 50x39x26 cm (32L). Märk den med en etikett från <a href="https://wiki.makerspace.se/Memberbooth">Memberbooth</a> och ställ den på en en fri plats på hyllplanen i korridoren. Lådan finns ex vis på <a href="https://www.clasohlson.com/se/F%C3%B6rvaringsl%C3%A5da-SmartStore-Home/34-1552-31">Clas Ohlson</a>. </p>
+<h3>Kom igång:</h3>
+<p>Först och främst, se till att du har allt du behöver för att komma igång:</p>
+<ul>
+    <li>Se till att du har laddat ner appen Accessy. Om du inte redan gjort detta så hittar du den information du behöver på <a href="https://wiki.makerspace.se/Accessy">Accessys wikisida</a>.</li>
+    <li>Se till att ladda ner och ordna ett konto på appen Slack (från <a herf="https://play.google.com/store/apps/details?id=com.Slack&hl=en_US&pli=1">Google Play</a> eller <a href="https://apps.apple.com/se/app/slack/id618783545">Apple Store</a>), eftersom det är det huvudsakliga sättet att kommunicera med andra på makerspacet. Om du behöver mer information, läs <a href="https://wiki.makerspace.se/Slack">Slacks wikisida</a></li>
+    <li>Here are links to other important online resources</li>
+    <ul>
+        <li>&#128100; <a href="https://medlem.makerspace.se/">Din medlemssida</a></li>
+        <li>&#x1F6D2; <a href="https://medlem.makerspace.se/shop">Medlemsshop</a></li>
+        <li>&#128198; <a href="https://www.makerspace.se/kalendarium">Kalendarium</a></li>
+        <li>&#128213; <a href="https://wiki.makerspace.se/V%C3%A4lkommen_till_Stockholm_Makerspace">Wikisidor</a></li>
+    </ul>
+    <li>Kolla gärna vad andra medlemmar har delat på <a href="https://stockholmmakerspace.slack.com/archives/C4M0MJ96K">#projekt</a>-kanalen på Slack och få inspiration!</li>
+    <li>Gå med i <a href="https://stockholmmakerspace.slack.com/archives/CGQ04052Q">#events</a>-kanalen på Slack för att få reda på kommande kurser, sammankomster och andra evenemang.</li>
```

Comment:
Det här borde vara default när man går med i slack. Om det inte är det så bör det ändras.

Reply:
Not sure what you mean here. Do you mean that by default Slack opens on the #events channel?

Reply:
People are automatically added to the follow channels when joining Slack:
![image](https://github.com/makerspace/makeradmin/assets/7572427/de2b1416-0e85-46a4-b2f7-81e683a52608)